src/ram/analysis/IndexAnalysis.h

```diff
@@ -324,8 +273,8 @@ class MinIndexSelection {
         return chainToOrder;
     }
 
-    /** @Brief check whether number of constraints in k is not equal
-        to number of columns in lexicographical order */
+    /** @Brief check whether number of constraints in k is not equal to number of columns in lexicographical
+     * order */
```

Comment:
The `@Brief` isn't needed here.
```suggestion
    /**
     * Check whether number of constraints in k is not equal to number of columns in lexicographical
     * order
     */
```